sdcm/sct_config.py

```diff
@@ -1361,7 +1361,7 @@ class SCTConfiguration(dict):
              type=boolean,
              help="Flag for running db node benchmarks before the tests"),
         dict(name="nemesis_selector", env="SCT_NEMESIS_SELECTOR",
-             type=str_or_list, k8s_multitenancy_supported=True,
+             type=str_or_list_or_eval, k8s_multitenancy_supported=True,
```

Comment:
why you need it eval ? in which use case ?

Reply:
In case when parameter will taken from env variable SCT_NEMESIS_SELECTOR